control/control_plane_core.go

```diff
@@ -247,6 +247,7 @@ func (c *controlPlaneCore) bindLan(ifname string, autoConfigKernelParameter bool
 	if autoConfigKernelParameter {
 		SetSendRedirects(ifname, "0")
 		SetForwarding(ifname, "1")
+		SetForwarding("all", "1")
```

Comment:
https://github.com/daeuniverse/dae/blob/main/docs/en/user-guide/kernel-parameters.md

Please update this doc.